Task1

Skurik-SA opened the pull request:
Добавил pgAdmin
В БД есть все 5 таблиц

Skurik-SA committed:
Task1

Добавил pgAdmin
В БД есть все 5 таблиц

chrn-feip commented on fefu-app2/docker-compose.yml:
```diff
@@ -40,7 +40,20 @@ services:
+        networks:
+            - sail
 networks:
+    app-tier:
```
Это зачем? У вас нет такой сети же в используемых контейнерами.

Skurik-SA replied:
После 20+ пересоздаваний проекта уже как-то выбилось из поля зрения

Skurik-SA committed:
Правки (Так это называть?)

Исправил, удалив эти строки.
Проект также запускается, только теперь без жёлтых предупреждений
